leetcode/src/61.c

```diff
@@ -0,0 +1,38 @@
+/**
+ * Given the head of a linked list, rotate the list to the right by k places.
+ *
+ *
+ * Definition for singly-linked list.
+ * struct ListNode {
+ *     int val;
+ *     struct ListNode *next;
+ * };
+ */
+struct ListNode* rotateRight(struct ListNode* head, int k){
+    struct ListNode *ptr = NULL, *prev =NULL;
+    ptr = head;
+    int count = 0;
+    while(ptr){
+        ptr = ptr->next;
+        count++;
+    }
+    if(count!=0)
+        k = k%count;
+    while(k--)
```

Comment:
Your solution essentially rotated the list right by one for k times, it would be more efficient to slice the list in two at the right location then stitch them back together.